pkg/controllers/tagging/tagging_controller.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
@nckturner do we need this in all files? Because some of the files are missing this so not sure if we need to add them there. @nguyenkndinh FYI.

Reply:
Good call out.  Please add them to the files you touch only, and we can add the rest in a separate PR.  Upstream kubernetes/kubernetes has a CI check for it, and we don't on this repo.